source/libsass.html.haml

```diff
@@ -37,11 +37,17 @@ title: libSass
   %li#go
     :markdown
       ### Go
-      [Wellington](https://github.com/wellington/wellington) is an extension to libsass that adds spriting and is available on brew:
-
-      `brew install wellington`
-
-      There are also three other libsass wrappers in go: [gosass](https://github.com/moovweb/gosass), [go-sass](https://github.com/SamWhited/go-sass) and [go_sass](https://github.com/suapapa/go_sass) which have not been updated in a while.
+      [go-libsass](https://github.com/wellington/go-libsass) has the most active
+      GoLang wrapper.  [gosass](https://github.com/moovweb/gosass) is another
+      libsass wrapper.
+      
+      [Wellington](http://getwt.io/) is an extension to libsass that adds
+      spriting.  It is available on brew: `brew install wellington`
+
+      [C6](https://github.com/c9s/c6) is a SASS 3.2 compatible implementation
```

Comment:
My take on c6 is that c6 doesn't aim to bring Sass to go. The author sees an opportunity to fork Sass and make a new, incompatible version of the language. The project lacks a compiler, so you can't do anything but create AST for the parts of this language that has been implemented and the main developer has discontinued work on it. Even if it was finished, there is no guarantee Sass code would compile with c6 and may never compile with it. It's interesting, but would be misleading to list it as a Sass compiler. Maybe if there was a toy project section or misc tools related to Sass section.